cli/version.go

```diff
@@ -49,7 +49,15 @@ func (v *VersionCommand) runVersion() error {
 }
 
 // versionExample shows examples in version command, and is used in auto-generated cli docs.
-// TODO: add example
 func versionExample() string {
-	return ""
+	return `$ pouch version
+GoVersion:       go1.9.1
+KernelVersion:
```

Comment:
Could you add a follow-up PR to make daemon side return KernelVersion?

Reply:
Well..I have spent some time on this. But I have not found the available library function to call.   Now the only way I can figure out is to open /proc/version on the user' computer and get the kernel information. I am not sure whether this is acceptable or not due to user privacy.

Reply:
I am afraid you should design a utils packages in ./pkg to make this.